### DISCUSSION TITLE
数据集下载地址

### DISCUSSION THREAD

**Discussion by jiangzhonglian** [apachecn/pytorch-doc-zh#588] [Show and tell]
大家可以给出数据集的下载地址

我抽空帮忙下载整理到一个地方，方便大家下载

**Comment by jiangzhonglian** [comment]
需要连接链接：

https://www.cs.toronto.edu/~kriz/cifar-10-python.tar.gz

**Comment by TianFengshou** [comment]
good job!

**Comment by liuzhaoo** [comment]
ucf101:
https://www.crcv.ucf.edu/data/UCF101/UCF101.rar

**Comment by liuzhaoo** [comment]
HMDB51
http://serre-lab.clps.brown.edu/wp-content/uploads/2013/10/hmdb51_org.rar

**Comment by liuzhaoo** [comment]
Kinetics-400,由于要在YouTube上下载，我找到了百度网盘版本：

链接: https://pan.baidu.com/s/1bCEGc2otJRyzG_CdmCnoQA 提取码: ed8h

**Comment by liuzhaoo** [comment]
Pascal VOC Dataset

voc2012:
train/val: http://pjreddie.com/media/files/VOCtrainval_11-May-2012.tar
test: http://pjreddie.com/media/files/VOC2012test.tar

voc2007:

train/val : http://pjreddie.com/media/files/VOCtrainval_06-Nov-2007.tar
test: http://pjreddie.com/media/files/VOCtest_06-Nov-2007.tar

**Comment by Jiuyong** [comment]
最好能附带下数据集的说明，有的数据集对于初学者来说，完全不知道是干啥的。

**Comment by algorithmlover2016** [comment]
> 需要连接链接：
> 
> https://www.cs.toronto.edu/~kriz/cifar-10-python.tar.gz

你好， 这个地址是不是用torchvision.datasets.CIFAR10 这个class 来下载有urlerror ，需要证书认证的错误？
https://github.com/apachecn/pytorch-doc-zh/blob/master/docs/1.4/8.md
ds_train = torchvision.datasets.CIFAR10(root=data_cifar10, train=True, download=True, transform=transform_train)

**Comment by algorithmlover2016** [comment]
> > 需要连接链接：
> > https://www.cs.toronto.edu/~kriz/cifar-10-python.tar.gz
> 
> 你好， 这个地址是不是用torchvision.datasets.CIFAR10 这个class 来下载有urlerror ，需要证书认证的错误？ https://github.com/apachecn/pytorch-doc-zh/blob/master/docs/1.4/8.md ds_train = torchvision.datasets.CIFAR10(root=data_cifar10, train=True, download=True, transform=transform_train)

error message:
SSLCertVerificationError                  Traceback (most recent call last)
SSLCertVerificationError: [SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: certificate has expired (_ssl.c:1129)

During handling of the above exception, another exception occurred:

**Comment by algorithmlover2016** [comment]
> > > 需要连接链接：
> > > https://www.cs.toronto.edu/~kriz/cifar-10-python.tar.gz
> > 
> > 
> > 你好， 这个地址是不是用torchvision.datasets.CIFAR10 这个class 来下载有urlerror ，需要证书认证的错误？ https://github.com/apachecn/pytorch-doc-zh/blob/master/docs/1.4/8.md ds_train = torchvision.datasets.CIFAR10(root=data_cifar10, train=True, download=True, transform=transform_train)
> 
> error message: SSLCertVerificationError Traceback (most recent call last) SSLCertVerificationError: [SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: certificate has expired (_ssl.c:1129)
> 
> During handling of the above exception, another exception occurred:

下载另外的数据集都没有问题，比如这个
torchvision.datasets.FashionMNIST
